Framework/Beamline/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-set(SRC_FILES 
-    src/ComponentInfo.cpp 
+set(SRC_FILES
+    src/ComponentInfo.cpp
```

Comment:
It looks like there is an indentation problem here.

Reply:
I think my editor just fixed the trailing whitespace from the line or are you referring to the indentation of the list?

Reply:
Sorry, it seems it was my editor messing up.